devel/make/Makefile

```diff
@@ -9,11 +9,11 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=make
 PKG_VERSION:=4.2.1
-PKG_RELEASE:=1
+PKG_RELEASE:=2
```

Comment:
Minor note: switching tarball format is not expect to cause change in final package content, so there is no need to bump `PKG_RELEASE` here.